src/app/authentication/pages/login/login.component.scss

```diff
@@ -0,0 +1,9 @@
+.content-wrapper {
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background-image: url('/assets/images/background.png');
```

Comment:
@carryall Panda can help us 🤣
![Screen Shot 2563-11-24 at 15 18 49](https://user-images.githubusercontent.com/749672/100067101-6d4a6280-2e68-11eb-8c6e-3cacd258603f.png)

Reply:
@Lahphim I tried compressed it before but the gradient doesn't look nice. The left one is the compressed one comparing to the original one on the right, what do you think?
<img width="1368" alt="Screen Shot 2563-11-25 at 10 16 23" src="https://user-images.githubusercontent.com/1772999/100178847-59a20900-2f07-11eb-8804-3737df7f42fc.png">

Reply:
@carryall I'm ok with the compressed one 🤔
Because it's too heavy for mobile.
If you feel like to keep it, I think about other such as dealing with this https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images

Reply:
@Lahphim  ✅  updated the image on [1ebfb09](https://github.com/carryall/survey-challenge/pull/23/commits/1ebfb09aaaa3262c18009cdba7c3e63b13424dc8)